Jenkinsfile

```diff
@@ -140,6 +140,30 @@ def daos_packages_version(String distro) {
     error "Don't know how to determine package version for " + distro
 }
 
+def fault_test_tag() {
+    if (skip_stage('fault-test')) {
+        return ",-faults"
+    }
+    return ""
+}
+
+def release_candidate() {
+    if (sh script: "git diff-index --name-only HEAD^ | grep -q TAG", returnStatus: true) {
+        if (sh script: "git describe --tags | grep -i 'rc'", returnStatus: true) {
+            return true
+        }
+    }
+    return false
+}
+
```

Comment:
Can I do this?

def release_candidate() {
    if env.ON_TAG {
        if env.RC_IN_TAG {
            return true
        }
    }
    return false
}
 and then add
       ON_TAG = sh script: "git diff-index --name-only HEAD^ | grep -q TAG", returnStatus: true
        RC_IN_TAG = sh script: "git describe --tags | grep -i 'rc'", returnStatus: true

Reply:
See above.